packages/lib/services/plugins/utils/manifestFromObject.ts

```diff
@@ -36,6 +36,15 @@ export default function manifestFromObject(o: any): PluginManifest {
 		return o.screenshots;
 	};
 
+	const getIcons = (defaultValue: Icons = null): Icons => {
```

Comment:
You're not using this defaultValue argument, so you can remove it

Reply:
Ok!